fabric/src/main/java/org/valkyrienskies/mod/fabric/mixin/compat/cc_restitched/MixinTurtleMoveCommand.java

```diff
@@ -42,16 +42,18 @@ public abstract class MixinTurtleMoveCommand {
     }
 
     //CUSTOM METHODS
+
     @Unique
     private static Vector3d getShipPosFromWorldPos(final Level world, final BlockPos position) {
```

Comment:
This logic is broken when multiple ships are intersecting the position. Can you implement the fix from https://github.com/ValkyrienSkies/Valkyrien-Skies-2/pull/1035?

Reply:
Implemented